spyder/widgets/mixins.py

```diff
@@ -130,66 +130,134 @@ def _update_stylesheet(self, widget):
                 }}'''.format(name, background, border)
             widget.setStyleSheet(css + extra_css)
 
-    def _format_text(self, title, text, color, ellide=False):
+    def _get_inspect_shortcut(self):
+        """
+        Queries the configuration for the inspect shortcut defined by the
+        editor.

```

Comment:
PEP 257 summary must be one line. We've been consistently enforcing this (even with Carlos)

``Queries the editor's config to get the current "Inspect" shortcut.``